DevAcademy/Scenes/Places/PlacesScene.swift

```diff
@@ -6,29 +6,55 @@
 //
 
 import SwiftUI
+import ActivityIndicatorView
 
 struct PlacesScene: View {
     @State var features: [Feature] = [] //Features.mock.features
+    @State var showFavorites = false
     
     var body: some View {
         NavigationStack {
             Group {
                 if !features.isEmpty {
                     List(features, id: \.properties.ogcFid) { feature in
-                        PlacesRow(feature: feature)
-                            .onTapGesture {
-                                onFeatureTapped(feature: feature)
-                            }
+                        NavigationLink {
+                            PlaceDetail(feature: feature)
+                        } label: {
+                            PlacesRow(feature: feature)
+//                                .onTapGesture {
+//                                    onFeatureTapped(feature: feature)
+//                                }
```

Comment:
Komentáře bych odstranil. Tuhle funkci aktuálně potřebovat nebudeme.
```
```

Reply:
ok, je to tam už zbytečné